vspace/src/main/webapp/WEB-INF/views/exhibition/space.jsp

```diff
@@ -42,6 +42,29 @@ function drawLinks() {
         $("#space").append(link);
     }
     </c:forEach>
+    
+    <c:forEach items="${externalLinkList}" var="link">
+    {
+        var posX = parseInt($("#space").css('margin-left')) + $("#space").position().left; 
+        var posY = $("#space").position().top;
+        var link = $('<a></a>');
+        link.attr('href', "${link.externalLink.externalLink}");
+        link.attr('target', "_blank");
+        
+        var linkDisplay = $('<span data-feather="navigation-2" class="fa fa-globe"></span>');
```

Comment:
did you notice that the icon is not shown? you need to remove the feather icon, or the globe icon won't show up.

Reply:
yes. I missed that. Thank you.